src/app/auth/duck/operations.ts

```diff
@@ -23,13 +20,13 @@ const fetchOauthMeta = clusterApi => {
   return async dispatch => {
     try {
       const res = await axios.get(oauthMetaUrl);
-      dispatch(setOauthMeta(res.data));
+      dispatch(Creators.setOauthMeta(res.data));
```

Comment:
I'm actually more in favor of this vs. aliasing them higher in the file; it makes the call very explicit as far as where it's coming from when reading the calling site.